File: atcoder.jp/contests/abc205/tasks/abc205_c/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const solve = (a: number, b: number, c: number) => {
  if (c % 2 == 0) {
    if (Math.abs(a) < Math.abs(b)) return "<";
    if (Math.abs(a) > Math.abs(b)) return ">";
    return "=";
  }
  if (a < b) return "<";
  if (a > b) return ">";
  return "=";
};

const [A, B, C] = input().split(" ").map(Number);
console.log(solve(A, B, C));

File: atcoder.jp/contests/abc205/tasks/abc205_c/main.py
A, B, C = map(int, input().split())


def solve() -> str:
    if 0 <= A and 0 <= B:
        if A < B:
            return "<"
        if A > B:
            return ">"
        return "="
    if C % 2 == 0:
        if abs(A) < abs(B):
            return "<"
        if abs(A) > abs(B):
            return ">"
        return "="
    if A < B:
        return "<"
    if A > B:
        return ">"
    return "="


print(solve())
